app/uploaders/avatar_uploader.rb

```diff
@@ -15,25 +11,30 @@ def store_dir
   end
 
   # Provide a default URL as a default if there hasn't been a file uploaded:
+
   def default_url
-    # For Rails 3.1+ asset pipeline compatibility:
-    # ActionController::Base.helpers.asset_path(
-    # "fallback/" + [version_name, "default.png"].compact.join('_')
-    # )
     "https://s3-us-west-2.amazonaws.com/tipsy-tally/default.jpg"
   end
 
   # Process files as they are uploaded:
-  # process :scale => [200, 300]
+  process scale: [200, 300]
   #
-  # def scale(width, height)
-  #   # do something
-  # end
+  def scale(width, height)
```

Comment:
Unused method argument - `width`. If it's necessary, use `_` or `_width` as an argument name to indicate that it won't be used. You can also write as `scale(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `height`. If it's necessary, use `_` or `_height` as an argument name to indicate that it won't be used. You can also write as `scale(*)` if you want the method to accept any arguments but don't care about them.